.github/workflows/static-analysis.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
-          php-version: 8.0
+          php-version: 8.1
```

Comment:
@nunomaduro is there any reason we shouldn't bump the "static analysis" GitHub action for the framework to run on PHP 8.1?

I've done it here as I use some PHP 8.1 syntax in the script.

I figure this is static analysis, so it may as well run alongside this one to save introducing another job.

Reply:
Unsure as Laravel v9 still relies on PHP 8.0 so maybe it's best to rely on the lower bound?

Static Analysis is probably the right workflow but I'd still opt for a different job as it's checking something differently. I'll work on it.